logger/proto/logger_config.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package logger;
+
+option go_package = "github.com/pokt-network/pocket/logger";
+
+message LoggerConfig {
+  // LogLevel level = 1;
+  // LogFormat format = 2;
+  string level = 1;
+  string format = 2;
+}
+
+enum LogLevel {
+  debug = 0;
+  info = 1;
+  warn = 2;
+  error = 3;
+  fatal = 4;
+  panic = 5;
+}
+
+enum LogFormat {
```

Comment:
If we decide to keep enums, please follow best practices: https://github.com/pokt-network/pocket/issues/258

Reply:
@Olshansk I don't mind removing them, but is there a place I can document possible config values? If no, I can start a config file documentation page.

Reply:
@Olshansk I've got familiar with the best practices doc, but I am not sure if they are suitable in this use-case.

My point of utilizing enubs for configuration is because we can specify potential static configuration values. No other parameters can be used but the ones defined in enum. If we follow the best practices document, then we need to do a couple of things:
1. rename the values: e.g., `debug` becomes `LOG_LEVEL_DEBUG`, which is not suitable as a value for that configuration parameter, we want `debug`. If we do rename `debug` to `LOG_LEVEL_DEBUG`, we'd need to then again do conversion from `LOG_LEVEL_DEBUG` to `debug` in `Create()`. Sounds very unnecesary.
2. introduce `UNSPECIFIED` value, which is not a valid value for configuration parameters.

Considering the values passed to the library - zerolog - I'd expect them to also do some verification so I'd ditch enums completely in case you want to enforce the enum best practices in that case. 

I totally agree - enums best practices make 100% sense when they are used for communication like grpc, but in this case this doesn't seem correct to me.

Reply:
Discussed offline, we will go with an enum for Pocket specific logging configs along with a mapper that maps the enums to `zerolog` specific strings. The following is just example pseudocode:

```
var pocketToZeroLogMappingConfig = map[EnumType]string{
	EnumType_LOG_LEVEL_DBEUG: "debug",
}
```